### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by peytonjo**
### What’s this PR do?
- Creates unit tests and begins creating Cypress tests for routing changes in the app 

### Where should the reviewer start?
- line 6 of  cypress/integration/examples/App_spec.js 
- line 71 of src/App/App.test.js
- line 1 of src/NavBar/NavBar.test.js 
### How should this be manually tested?
- run "npm test" or open cypress on the deployed site

### CODE REVIEW

File: src/NavBar/NavBar.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const NavBar = (props) => {
       </div>
       <section className="nav-btns">
       <NavLink to="/user-view">
-        <button alt="button">
+        <button role="button">
```
Comment (jdxsmith) [inline src/NavBar/NavBar.js:14]: This role could be more speific

File: src/User/User.js [hunk 2]
```diff
@@ -12,13 +12,13 @@ class User extends Component {
   render() {
    return (
       <section className="user-view">
-        <article className="user-favs">
+        <article className="user-favs" alt="favorite jokes">
```
Comment (jdxsmith) [inline src/User/User.js:15]: What is the purpose of these alt tags?  I don't see the need for them on an article

**Review by jdxsmith — APPROVED** [review]
Nice work getting some more testing done!  I still think there is more testing to do in order to meet 90% coverage,  but we can do that once our app has more functionibility

File: cypress/integration/examples/App_spec.js [hunk 3]
```diff
@@ -3,8 +3,22 @@ describe('App', () => {
...
+  it('should render users favorited jokes when clicked', () => {
+    cy.get('[href="/user-view"] > button').click()
+    cy.url().should('include', 'user-view')
+  });
+
+  it('should render users created jokes when clicked', () => {
+    cy.get('[href="/user-view"] > button').click()
+    cy.get('.users-jokes > .joke-card')
+  });
+
```
Comment (CalebCyphers) [inline cypress/integration/examples/App_spec.js:23]: We should build some tests that follow specific user stories. They can be longer and more complex

**Review by CalebCyphers — APPROVED** [review]

**Pull request merged**